src/domain/event/enrol/Enrol.tsx

```diff
@@ -36,7 +38,17 @@ const Enrol: FunctionComponent = () => {
   // Might need to refetch myProfile in any case
   const [enrolOccurrence] = useMutation<EnrolOccurrenceMutationInput>(
     enrolOccurrenceMutation,
-    {}
+    {
+      refetchQueries: [
+        { query: profileQuery },
+        {
+          query: childByIdQuery,
+          variables: {
+            id: params.childId,
+          },
+        },
+      ],
+    }
```

Comment:
I'm unsure should `<Enrol />` be concerned about these.

Reply:
I agree. But with our current implementation the UI will be out of date from the server without this.